crates/consensus/src/transaction/eip4844.rs

```diff
@@ -653,6 +653,19 @@ impl TxEip4844 {
         // 'transaction type byte length' + 'header length' + 'payload length'
         1 + Header { list: true, payload_length }.length() + payload_length
     }
+
+    /// Output the length of the RLP signed transaction encoding. This encodes with a RLP header.
+    pub fn payload_len_with_signature(&self, signature: &Signature) -> usize {
+        let len = self.payload_len_with_signature_without_header(signature);
+        length_of_length(len) + len
+    }
+
+    /// Output the length of the RLP signed transaction encoding, _without_ a RLP header.
+    pub fn payload_len_with_signature_without_header(&self, signature: &Signature) -> usize {
```

Comment:
same q

Reply:
I think it doesn't hurt to make the methods more generic, just modified